lib/ui/settings/SettingsHomePanel.dart

```diff
@@ -912,18 +979,25 @@ class _SettingsHomePanelState extends State<SettingsHomePanel> implements Notifi
   // Privacy
 
   Widget _buildPrivacy() {
-    return _OptionsSection(
-      title: Localization().getStringEx("panel.settings.home.privacy.title", "Privacy"),
-      widgets: <Widget>[
-        RibbonButton(
+    List<Widget> contentList = new List();
+
+    List<dynamic> codes = FlexUI()['settings.privacy'] ?? [];
+    for (int index = 0; index < codes.length; index++) {
+      String code = codes[index];
+      if (code == 'statement') {
+        contentList.add(RibbonButton(
           height: null,
-          borderRadius: _borderRadiusFromIndex(0, 1),
```

Comment:
@TodorBachvarov Please review this change is correct. It's related to a previous fix made by you. Awaiting for confirmation